stories/autocomplete.stories.ts

```diff
@@ -8,7 +10,7 @@ storiesOf('Basics|Autocomplete', module).add(
 
     container.appendChild(instantSearchAutocomplete);
 
-    const customAutocomplete = instantsearch.connectors.connectAutocomplete(
+    const customAutocomplete = connectAutocomplete<{ container: HTMLElement }>(
```

Comment:
how come container isn't inferred?

Reply:
@Haroenv There is no container on the connector, only on the widget. Here you have a custom widget, therefore you need to pass your widgets types.